project_recalculate/wizard/recalculate_wizard.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, fields, api
+from openerp.exceptions import Warning
+from openerp.tools.translate import _
+
+
+class ProjectRecalculateWizard(models.TransientModel):
+    _name = 'project.recalculate.wizard'
+
+    project = fields.Many2one(comodel_name='project.project', readonly=True)
```

Comment:
Just checked and it is in the guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field